tests/src/Functional/NewsroomConfigurationTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_newsroom\Functional;
+
+use Drupal\oe_newsroom\OeNewsroom;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Test the Newsroom API configuration.
+ *
+ * @group oe_newsroom_newsletter
+ */
+class NewsroomConfigurationTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_newsroom',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Test if the Newsroom factory retrieves the correct configuration values.
```

Comment:
We have no factory anymore.
```suggestion
   * Test the Newsroom configuration page.
```